lib/engine/game/g_18_oe/game.rb

```diff
@@ -267,6 +501,13 @@ def after_par(corporation)
           corporation.spend(TRACK_RIGHTS_COST[region], @bank)
         end
 
+        def issuable_shares(entity)
+          return [] if !entity.corporation? || (entity.corporation? && entity.type != 'major')
```

Comment:
Again, check for `:major` not `'major'`.
The `entity.corporation?` check on the right-hand side of the operator is redundant.